client/src/shared/utils/interface.ts

```diff
@@ -159,3 +164,45 @@ export interface Learner {
   firstname: string;
   lastname: string;
 }
+
+export interface CourseLesson {
+  id: number;
+  title: string;
+  link: string;
+  description: string;
+  is_active: boolean;
+  created_at: string;
+  updated_at: string;
+  course: number;
```

Comment:
I think there is already an existing `Lesson` interface in this file that should meet your needs, please try to utilize that, and do we need `is_active` and `course` values in FE? , please let me know

Reply:
is_active and course is not actually required in my task, I just add it to much the API response, but Ill just use the existing Lesson interface